### PR TITLE
rabbitmq_ampqlib_queue

### PR DESCRIPTION
**Pull request by thisboyiscrazy**
Allows messages to be delivered to multiple queues based on rctp_to

### CODE REVIEW

File: plugins/queue/rabbitmq_amqplib_queue.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
+// queue/rabbitmq_amqplib
+
+var amqp = require("amqplib/callback_api");
+
+var channel;
+var deliveryMode;
+var logger = require('./logger');
```
Comment (msimerson) [inline plugins/queue/rabbitmq_amqplib_queue.js]: logger is unused and should be removed.

**Comment by msimerson** [comment]
My gut feel is that instead of being a new plugin, this would be better as a feature addition of the existing `queue/rabbitmq_ampqlib` plugin. Add a config setting to rabbirmq_amqplib.ini and when enabled, it registers the *data* and *queue_outbound* hooks, enabling this feature.

**Comment by msimerson** [comment]
If you have objections to integrating this with the existing plugin, then this PR really should include a paragraph in both plugins explaining why they exist separately, with references to each other and guidance for choosing the most appropriate one.

**Comment by thisboyiscrazy** [comment]
That's is what i was originally going, I will do that.

**Comment by msimerson** [comment]
closing, in expectation of a future PR that merges this with the existing rabbitmq plugin.

**Pull request closed without merging**